### ISSUE TITLE
Why use getattr here?

### ISSUE THREAD

**Issue by trewjames** [mineugene/spot-rename#3]
```
def eventloop_exec(self):
        try:
            app = getattr(self, "_app")
            logger.debug("Starting event-loop...")
            app.exec_()
```

why not just
```
def eventloop_exec(self):
        try:
            logger.debug("Starting event-loop...")
            self._app.exec_()
```

**Comment by mineugene** [comment]
Yes, both are completely valid calls in this case. It was possibly made more clear using `getattr` that the instance attribute `_app` does not exist in some cases (program execution using the `--nogui` flag). Hence, the check for a raised `AttributeError` later.

For instance, here is an example of when it is assumed to exist for every case (mvc.view.widget_display.py):
```
def invoke_increment(self):
    self.callback.ping()
```
I did not do `getattr(self.callback, "ping")` here as `ping` is expected to always exist, although it is dynamically bound in `View.__init__`.


### Extra

It is expected that `ping` always exists, because in `Controller.__init__`, the `handle` kwarg is passed unconditionally. But then one would argue why `View.__init__` has a conditional statement for checking if `handle` kwarg exists. This is because an error would occur when one of the child classes of `View`, such as `WidgetDisplay` makes a `super().__init__()` call that runs `View.__init__` again without any arguments. Thus, all conditionals are false in this case.

This is visible when the `--debug_enable` flag is used. View is initialized twice.
```
2020-08-09 04:06:49,127 [ mvc.view.mode | DEBUG ] Init; received fwd args {'nogui': True, 'handle': <mvc.controller.interface.Interface object at 0x02199550>}, ([],)
2020-08-09 04:06:49,127 [ mvc.view.mode | DEBUG ] Init; received fwd args {}, ()
```
